package.json

```diff
@@ -12,15 +12,24 @@
   "bugs": {
     "url": "https://github.com/RecordReplay/protocol/issues"
   },
+  "files": [
+    "js",
+    "json",
+    "LICENSE",
+    "README.md"
+  ],
   "main": "js/index.js",
   "scripts": {
     "build-json": "./build.sh",
     "build-ts": "ts-node build",
     "build-js": "tsc",
-    "build": "npm run build-json && npm run build-ts && npm run build-js"
+    "build": "npm run build-json && npm run build-ts && npm run build-js",
```

Comment:
where would we want to call `clean`? perhaps in prepare?

Reply:
I only added `clean` as a convenience for the user, to get rid of the generated files if he doesn't want to leave them lying around. It's not necessary for the build process.

Reply:
got it

Reply:
I now added `clean` as the first build step to ensure that if we decide to rename a file, the old generated files are removed.